lib/taiko.js

```diff
@@ -2449,66 +2454,96 @@ module.exports.comboBox = (attrValuePairs, ...args) => {
 module.exports.dropDown = dropDown;
 
 function dropDown(attrValuePairs, ...args) {
-    validate();
-    const {selector} = prepareParameters(attrValuePairs, ...args);
-    const get = getElementGetter(selector,
-        async () => await $$xpath(`//select[@id=(//label[contains(string(), ${xpath(selector.label)})]/@for)] | //label[contains(string(), ${xpath(selector.label)})]/select`),
-        '//select');
-    return {
-        get: getIfExists(get),
-        exists: exists(getIfExists(get), descEvent),
-        description: desc(selector, 'label', 'Dropdown'),
-        select: async (value) => {
-            const nodeId = (await getIfExists(get)())[0];
-            if (!nodeId) throw `${desc(selector, 'label', 'DropDown')} not found`;
-            if (defaultConfig.headful) await highlightElemOnAction(nodeId);
-
-            function selectBox(value) {
-                let found_value = false;
-                if(this.options.length > value.index){
-                    this.selectedIndex = value.index;
-                    found_value = true;
-                    let event = new Event('change');
-                    try{
-                        this.dispatchEvent(event);
-                    }catch(e){
-                        return {'error': 'Error occured while dispatching event','stack': e.stack};
-                    }
-                    return found_value;
-                }
-                for (var i = 0; i < this.options.length; i++) {
-                    if (this.options[i].text === value || this.options[i].value === value) {
-                        this.selectedIndex = i;
-                        found_value = true;
-                        let event = new Event('change');
-                        try{
-                            this.dispatchEvent(event);
-                        }catch(e){
-                            return {'error': 'Error occured while dispatching event','stack': e.stack};
-                        }
-                        break;
-                    }
-                }
-                return found_value;
+  validate();
```

Comment:
I think this is just a lint change. Can we not commit it with this PR?

Reply:
Agree!